### DISCUSSION TITLE
Field Textarea is not displayed in Stack

### DISCUSSION THREAD

**Discussion by proart** [laravel/nova-issues#6442] [Q&A]
- Laravel Version: 8.83.27
- Nova Version: 4.33.2
- PHP Version: 8.1

### Description:
Hi,
Nova doesn't display "Textarea" field in "Stack" field on "Details" page:

### Detailed steps to reproduce the issue on a fresh Nova installation:
Put this method in some resource and check details page.

```php
use Laravel\Nova\Fields\Stack;
use Laravel\Nova\Fields\Text;
use Laravel\Nova\Fields\Textarea;
use Laravel\Nova\Http\Requests\NovaRequest;

// ...

public function fieldsForDetail(NovaRequest $request)
{
    return [
        Stack::make('Stack field', [
            Text::make('test 0', fn() => 'text field 0'),

            Textarea::make('test TEXAREA', fn() => '!!! WHERE IS THIS TEXT? !!!')->alwaysShow(),

            Text::make('test 1', fn() => 'text field 1'),

            Text::make('test 2', fn() => 'text field 2'),
        ])
    ];
}
```
Result:
![image](https://github.com/laravel/nova-issues/assets/1822794/9a6d998b-35fe-4b01-a90e-56c19e71278b)

How to display the field Textarea?

**Comment by webard** [comment]
I think Textarea is not meant to be in Stack field. It's main use case is to organize data in table, where textarea is hidden too.
